src/quo/components/settings/settings_item/view.cljs

```diff
@@ -61,6 +67,7 @@
        :avatar      [user-avatar/user-avatar image-props]
        :icon-avatar [icon-avatar/icon-avatar image-props]
        :token       [token/view image-props]
+       :emoji       [emoji-component image-props]
```

Comment:
As emoji is actually a char and not an image, could we decouple it from the `image-component` component?

Reply:
I find it a bit inconsistent to have an `image-props` property that takes text. While it would work, having a top-level key like :emoji, as you mentioned, seems more intuitive and makes more sense. That said, it's not a strong opinion or a blocker, just a minor observation.

Reply:
One more consideration - in Figma `settings-item` component also designed in a way that `:emoji` is one of the values of `image` property. So If you aren't strongly against, I'd be glad to leave implementation as is, at least for the sake of alignment with design 🙏 
<img width="733" alt="Screenshot 2024-11-21 at 12 37 25" src="https://github.com/user-attachments/assets/9e8fb9b5-4f6a-47a2-8123-1e3e000ebab1">